CONTRIBUTING.md

```diff
@@ -1,62 +1,21 @@
 # Contributing
-Pull requests are welcome, but please
-[make an issue](https://github.com/Kneasle/ringing-room-bot/issues/new) to discuss the changes
-before starting to implement things.
 
-To run Wheatley **from source code**, `cd` to the repository directory and run:
-```bash
-python3 run-wheatley [ARGS]
-```
-(or `python run-wheatley [ARGS]` on Windows).
+**First**: if you're unsure or afraid of _anything_, just ask or submit the issue or pull request
+anyway.  You won't be yelled at for giving it your best effort.  The worst that can happen is that
+you'll be politely asked to change something.  We appreciate any sort of contributions, and don't
+want a wall of rules to get in the way of that.
 
-Or, on Unix you can run `./run-wheatley [ARGS]`.
+If you're new to the code, a high-level description of Wheatley's internal architecture can be found
+in [ARCHITECTURE.md](https://github.com/kneasle/wheatley/blob/better-contributing/ARCHITECTURE.md).
 
-## Code structure
+## Testing Your Changes
+
+To run Wheatley **from source code**, `cd` to the repository root and run:
```

Comment:
Consider adding a getting started with the repo
 - Fork & clone with git
 - (Optional) Create & activate a [virtual environment](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/);
 - Install python dependencies `pip install -r requirements.txt`

Reply:
Added!